ACE/debian/copyright

```diff
@@ -1,141 +1,251 @@
-This package was debianized by:
- Ossama Othman <ossama@debian.org> on Fri, 25 Sep 1998 16:43:42 -0400.
-Then maintained by:
- Brian Nelson <pyro@debian.org>
- Konstantinos Margaritis <markos@debian.org>
- Thomas Girard <thomas.g.girard@free.fr>
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ACE
+Upstream-Contact: Johnny Willemsen <jwillemsen@remedy.nl>
+Source: https://github.com/DOCGroup/ACE_TAO
+Files-Excluded: ASNMP
+                performance-tests
+                rpmbuild debian
+                websvcs
+                bin/LabVIEW_RT/*.exe
+                examples/C++NPv2
+                examples/C++NPv1
+                examples/APG
+                contrib
+                apps/JAWS/clients/WebSTONE
+                MPC/MPC.ico
+                apps/gperf/tests/test-*.exp
+                docs/ACE-subsets.html
+                docs/ACE-guidelines.html
 
-It is now maintained by:
- Debian ACE+TAO maintainers <pkg-ace-devel@lists.alioth.debian.org>
+Files:     *
+Copyright: 1993-2019 Douglas C. Schmidt and his research group at
+           Washington University, University of California, Irvine,
+           and Vanderbilt University
+License:   other-BSD
```

Comment:
There is a formal DOC license, see https://spdx.org/licenses/DOC

Reply:
Great. I will change the name in the next Debian upload.